src/lib/autoinstall/clients/inst_autosetup.rb

```diff
@@ -386,6 +386,11 @@ def main
         #
         autosetup_firewall
 
+        Progress.NextStage
+
+        # Validate the security policies
+        autosetup_security_policies unless Yast::AutoinstConfig.Confirm
```

Comment:
What? So if I disable the confirm mode it will skip the policies even if configured in the profile?

Reply:
I guess the installation summary (which includes a proposal for the policies) is shown when confirm is enabled. So, as fas as I undestand it:

* If confirm is enabled, then the policies are checked by the policies proposal in the installation summary.
* If confirm is disabled, then the policies are checked by `#autosetup_security_policies`. 
* If `#autosetup_security_policies` finds a failing rule, then the confirm mode is automatically enabled and the user can enable/disable/fix rules.
* If `#autosetup_security_policies` does not find failing rules, then the autoinstallation continue without confirm.

@imobachgs is this correct?

Reply:
BTW, I agree with explaining this with a comment in the code.

Reply:
Yes, it is correct. The idea is to fall back to the confirmation screen if the policies are not passing. It is the same approach that we follow when we cannot "calculate" an upgrade.